scripts/build_lbann.sh

```diff
@@ -38,7 +38,7 @@ CMD_LINE_VARIANTS=
 
 # Default versions of Hydrogen, DiHydrogen, and Aluminum - use head of repo
 HYDROGEN_VER="@develop"
-ALUMINUM_VER="@1.2.1"
+ALUMINUM_VER="@1.2.3:"
```

Comment:
what does the trailing colon do?

Reply:
"Any version newer than". (I had to ask the same question. 😉 )

Reply:
Yes that is correct.